src/main/java/com/puppycrawl/tools/checkstyle/checks/UniquePropertiesCheck.java

```diff
@@ -67,14 +67,7 @@ protected void processFiltered(File file, FileText fileText) {
         final UniqueProperties properties = new UniqueProperties();
 
         try {
-            final FileInputStream fileInputStream = new FileInputStream(file);
-            try {
-                // As file is already read, there should not be any exceptions.
-                properties.load(fileInputStream);
-            }
-            finally {
-                fileInputStream.close();
-            }
+            loadProperties(file, properties);
```

Comment:
you need to move fileInputStream out of TRY scope and merge TRYs, so finally become part of top TRY.